content/events/IPFS-friends-ethcc.md

```diff
@@ -0,0 +1,26 @@
+---
+eventName: "IPFS + Friends Cafe: Paris" #the name of your track or event and its mandatory
+
+eventDescription: "​Cafes emerged in Europe in the 17th century, serving as a place to foster community and promote the exchange of ideas and philosophy. Paris also welcomed in this Age of Englightenment.
+
+​Fitting for us, we'd like to invite you to join us in Paris during EthCC at the IPFS & Friends Cafe to discuss or present anything IPFS or Dweb related.
+
+Our venue is not yet confirmed but we are looking at July 17." #short description of your track or event limiting to 100-150 characters
+
+location: "Paris, France" or "Location TBD" #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
```

Comment:
@laeccles Please remove the comments as its better suggested. Also update the location value to either one of the options: 'Paris, France' or 'Location TBD'

Reply:
@WinsCodes I have removed the host and logo while I confirm this info.  Can we add this to the calendar and I will create a pull request when I have the host and logo confirmed?